ewmh.c

```diff
@@ -228,6 +228,9 @@ ewmh_init(void)
     luaA_class_connect_signal(L, &client_class, "property::titlebar_left" , ewmh_client_update_frame_extents);
     luaA_class_connect_signal(L, &client_class, "property::border_width" , ewmh_client_update_frame_extents);
     luaA_class_connect_signal(L, &client_class, "manage", ewmh_client_update_frame_extents);
+    /* NET_CURRENT_DESKTOP handling */
+    luaA_class_connect_signal(L, &client_class, "property::screen", ewmh_update_net_current_desktop);
```

Comment:
I don't think that this needs to connect to `property::screen`. The function `ewmh_update_net_current_desktop` does not use a client's screen. However, the `tagged` and `untagged` signals on `client_class` are necessary (if the currently focused client is moved to another tag).